src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -1011,20 +1016,22 @@ private static string GetInputTypeString(InputType inputType)
             }
         }
 
-        private static IEnumerable<SelectListItem> GetSelectListItems([NotNull] ViewContext viewContext, string name)
+        private static IEnumerable<SelectListItem> GetSelectListItems(
+            [NotNull] ViewContext viewContext,
+            string expression)
         {
-            var value = viewContext.ViewData.Eval(name);
+            var value = viewContext.ViewData.Eval(expression);
             if (value == null)
             {
                 throw new InvalidOperationException(Resources.FormatHtmlHelper_MissingSelectData(
-                    "IEnumerable<SelectListItem>", name));
+                    "IEnumerable<SelectListItem>", expression));
```

Comment:
we're never going to change `SelectListItem`'s name but, sure, I'll throw in a gratuitous use or two of C# 6 :smile_cat: